CHANGELOG.md

```diff
@@ -99,6 +115,15 @@ You can participate in the conversation around this release in [this issue](ADD)
 - Make SafeAreView flow strict-local ([84915a2d68](https://github.com/facebook/react-native/commit/84915a2d68c2cb4cc17ca3a4651f33ba71a787ee) by [@TheSavior](https://github.com/TheSavior))
 - Strengthen flowtype of measureLayout ([bcc58bce02](https://github.com/facebook/react-native/commit/bcc58bce02acf51f6d5cff6e5457f447f12b1e30) by [@TheSavior](https://github.com/TheSavior))
 - Upgrade to latest dependencies in package.json template ([babf67f11c](https://github.com/facebook/react-native/commit/babf67f11c09397b11fa1dd8e82e3b5edddb8c25) by [@codler](https://github.com/codler))
+- Enable no-useless-escape lint rule ([90977b0e00](https://github.com/facebook/react-native/commit/90977b0e00acc6b3263502017c27094392e89478) by [@motiz88](https://github.com/motiz88))
+- Support removeClippedSubviews for horizontal ScrollViews ([42152a3fa3](https://github.com/facebook/react-native/commit/42152a3fa3f949f5112461753eb44a436355dfb1))
+- Update DevSettings.reload to accept a reason ([549cac63cb](https://github.com/facebook/react-native/commit/549cac63cb252037f73453c5d4e7ae5f15586607) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Upgrade eslint-plugin-relay to 1.3.12 ([f0bcfbe9be](https://github.com/facebook/react-native/commit/f0bcfbe9be0eb6a06d096a682717a23e43c39d52) by [@jstejada](https://github.com/jstejada))
+- Make setting useNativeDriver required. Add runtime warning if not specified ([5876052615](https://github.com/facebook/react-native/commit/5876052615f4858ed5fc32fa3da9b64695974238) by [@TheSavior](https://github.com/TheSavior))
```

Comment:
This should probably be categorized as a deprecation as it will eventually become an error in a future release